services/coreos/rbac.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
This will need to be an admin resource, I imagine.

Reply:
I don't think so - these are [regular RoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)s scoped to the `coreos` namespace, not `ClusterRoleBindings`.

Reply:
OK. Our general pattern has been to consider _any_ RBAC to be admin so it's got a different workflow, but feel free to do whatever you want under `services/`

Reply:
Tricky topic; feels like we want to cleanly separate things which are *cluster* admin vs just project RBAC?  I kept it as is, hope that's ok.